### ISSUE TITLE
[Feature Request] Timelapse Rendering Completion

### ISSUE THREAD

**Issue by ralxnder** [fabianonline/OctoPrint-Telegram#120]
I'm running on an older Pi and I get the notification that I shouldn't start another print job until after the rendering has completed. However, if the print job was very large the rendering could take quite a while. It would be nice to be able to have Telegram send me a message when the rendering is done so that I don't have to keep checking on it.

**Comment by vbz89** [comment]
Would there also be a way to get the telegram bot to upload the time lapse?

Knowing that the time-lapse has ended would definitely be very useful.

**Comment by ralxnder** [comment]
Uploading the time lapse would be nice to have but as an enhancement.
